XCUITests/FxScreenGraph.swift

```diff
@@ -38,28 +38,86 @@ let allSettingsScreens = [
     OpenWithSettings,
 ]
 
+let Intro_Organize = "Intro.Organize"
+let Intro_Customize = "Intro.Customize"
+let Intro_Share = "Intro.Share"
+let Intro_Choose = "Intro.Choose"
+let Intro_Sync = "Intro.Sync"
+
+let allIntroPages = [
+    Intro_Organize,
+    Intro_Customize,
+    Intro_Share,
+    Intro_Choose,
+    Intro_Sync,
+]
+
+let HomePanelsScreen = "HomePanels"
+let HomePanel_TopSites = "HomePanel.TopSites.0"
+let HomePanel_Bookmarks = "HomePanel.Bookmarks.1"
+let HomePanel_History = "HomePanel.History.2"
+let HomePanel_ReadingList = "HomePanel.ReadingList.3"
+
+let allHomePanels = [
+    HomePanel_Bookmarks,
+    HomePanel_TopSites,
+    HomePanel_History,
+    HomePanel_ReadingList,
+]
+
+let ContextMenu_ReloadButton = "ContextMenu_ReloadButton"
+
 func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozilla.org/en-US/book/") -> ScreenGraph {
-    let map = ScreenGraph(app)
+    let map = ScreenGraph()
 
+    let startBrowsingButton = app.buttons["IntroViewController.startBrowsingButton"]
     map.createScene(FirstRun) { scene in
         scene.gesture(to: NewTabScreen) {
-            let firstRunUI = app.buttons["Start Browsing"]
-            if firstRunUI.exists {
-                firstRunUI.tap()
+            if startBrowsingButton.exists {
+                startBrowsingButton.tap()
             }
         }
+
+        scene.noop(to: allIntroPages[0])
+    }
+
+    // Add the intro screens.
+    var i = 0
+    let introLast = allIntroPages.count - 1
+    let introPager = app.scrollViews["IntroViewController.scrollView"]
+    for intro in allIntroPages {
+        let prev = i == 0 ? nil : allIntroPages[i - 1]
+        let next = i == introLast ? nil : allIntroPages[i + 1]
+
+        map.createScene(intro) { scene in
```

Comment:
> `Settings screen -> Show tour`
Yes, that was possible once as `SettingsScreen` -> `FirstRun`; not sure where that ended up. Feel free to add it when you need to.

> Adding `FxSyncLogin` page.
Yes that makes sense.

>